src/main/flight/rpm_filter.c

```diff
@@ -224,4 +224,16 @@ bool isRpmFilterEnabled(void)
     return (motorConfig()->dev.useDshotTelemetry && (rpmFilterConfig()->gyro_rpm_notch_harmonics || rpmFilterConfig()->dterm_rpm_notch_harmonics));
 }
 
+float rpmMinMotorSpeed()
+{
+    float minSpeed = 10000.0f;
+    for (int i = getMotorCount(); i--;) {
```

Comment:
It might be more efficient to cache the result of the calculation, and either invalidate or recalculate it when `motorFrequency` is updated.

Also, using `minSpeed` to store `MIN(motorFrequency)` seems to be a misnomer.

Reply:
I fixed the misnomers. Good point rg avoiding calculation of minMotorFrequency multiple times. I've also addressed that.